lib/stormpath-sdk/resource/collection.rb

```diff
@@ -51,6 +51,10 @@ def limit limit
     self
   end
 
+  def size
+    current_page.items.size
```

Comment:
The size should not be retrieved from the items list, it is a property. This should be a property of the `CollectionPage` class, just like offset, limit and items are.

So, this method shouldn't be necessary to keep it consistent with the other properties that are already retrieved from the current page.